storage-service/README.md

```diff
@@ -8,22 +8,7 @@ In the future the storage-service can be extended to support other storage types
 
 ## Environment Variables
 
-To ensure all necessary environment variables are set correctly this section describes all environment variables across
-all services.
-
-### Storage-Service
-
-| Env Variable                | Default Value | Description                                                                                     |
-| --------------------------- | ------------- | ----------------------------------------------------------------------------------------------- |
-| STORAGE_SERVICE_HOST        | localhost     | IP address of storage service                                                                   |
-| PORT                        | 8090          | The port used to expose the storage service                                                     |
-| ACCESS_CONTROL_ALLOW_ORIGIN | "\*"          | CORS configuration                                                                              |
-| MINIO_ACCESS_KEY            | minio         | Access key for Minio server                                                                     |
-| MINIO_SECRET_KEY            | minio123      | Secret (Password) for Minio server                                                              |
-| MINIO_PORT                  | 9000          | Port of connected Minio                                                                         |
-| MINIO_HOST                  | localhost     | IP address of connected Minio server                                                            |
-| MINIO_BUCKET_NAME           | trubudget     | Bucket name of the connected Minio server                                                       |
-| LOG_LEVEL                   | info          | Defines the log output. Supported levels are `trace`, `debug`, `info`, `warn`, `error`, `fatal` |
+See [Environment Variables](./environment-variables.md)
```

Comment:
Same as the previous README files, remove deprecated things.

Also [this section](https://github.com/openkfw/TruBudget/blob/c3a0c7daefa3d12ed73b78880340d400352ef57e/storage-service/README.md#start-without-docker) feels a little unncessary to me - do we need to mention how to start every component? we can just say start blockchain, api and so on and then just mention how the storage service is started.